Porting.md

```diff
@@ -167,7 +167,7 @@ mka hybris-boot
 mka systemimage
 ```
 
-Here is also recomended to set `-j[num]` to do parallel building, this reduces build time. *NOTE* this is not needed on halium-7.1 since it will calculate parallel building based on CPU performance.
+If you use `make` and not `mka` it's recommended to set `-j[num]` to do parallel building, this reduces build time. *NOTE* this is not needed on halium-7.1 since it will calculate parallel building based on CPU performance.
```

Comment:
Then remove the halium-7.1 note? 😉  I don't think the behavior of make has changed